aspnetcore/signalr/configuration.md

```diff
@@ -325,6 +325,52 @@ var connection = new signalR.HubConnectionBuilder()
 
 ---
 
+### Configure stateful reconnect
+
+SignalR stateful reconnect reduces the perceived downtime of clients that have a temporary disconnect in their network connection, such as when switching network connections or a short temporary loss in access.
+
+Stateful reconnect achieves this by:
+
+* Temporarily buffering data on the server and client.
+* Acknowledging messages received (ACK-ing) by both the server and client.
+* Recognizing when a connection is returning and replaying messages that might have been sent while the connection was down.
+
+Stateful reconnect is available in ASP.NET Core 8.0 and later.
+
+Opt-in to stateful reconnect at both the server hub enpoint and the client:
+
+* Update the server hub endpoint configuration to enable the `AllowStatefulReconnects` option:
+
+  ```csharp
+  app.MapHub<MyHub>("/hubName", options =>
+  {
+      options.AllowStatefulReconnects = true;
+  });
+  ```
```

Comment:
I forgot that the server can also configure its max buffer size.
For global options:
```csharp
builder.AddSignalR(o => o.StatefulReconnectBufferSize = 1000);
```
or for hub specific options:
```csharp
builder.AddSignalR().AddHubOptions<MyHub>(o => o.StatefulReconnectBufferSize = 1000);
```